misk-jdbc/src/main/kotlin/misk/jdbc/DataSourceService.kt

```diff
@@ -113,6 +116,33 @@ class DataSourceService @JvmOverloads constructor(
       hikariConfig.dataSourceProperties["characterEncoding"] = "UTF-8"
     }
 
+    /*
+      TODO(sahilm): The same mitigation might be applicable for Vitess.
+      Similar looking issues from internal Cash App slack for Vitess backed service:
+        * https://cash.slack.com/archives/C06D6QEL9B2/p1704403704775189
```

Comment:
Let's not include internal links within the OSS source code please. Better to include this in an internal task tracker.